packages/css-data/bin/mdn-data.ts

```diff
@@ -57,12 +37,62 @@ const normalizedValues = {
     value: 0,
     unit: "px",
   },
-  // https://github.com/mdn/data/issues/556
-  // @todo remove once fixed
   "background-size": autoValue,
   "text-size-adjust": autoValue,
 } as const;
 
+const parseInitialValue = (property: string, value: string): StyleValue => {
+  // Our default values hardcoded because no single standard
+  if (property in normalizedValues) {
+    return normalizedValues[property as keyof typeof normalizedValues];
```

Comment:
why do you need an assertion here? its a string by default and you already checked its in the object, I don't see a need to assert to the compiler its a specific property

Reply:
🤷